distribution/docs/src/main/jdocs/content/_plugins/video-thumbnail-plugin-contents.adoc

```diff
@@ -0,0 +1,25 @@
+:type: plugin
+:status: published
+:title: Video Thumbnail Plugin
+:link: _video_thumbnail_plugin
+:plugintypes: postcreatestorage, postupdatestorage
+:summary: Generates thumbnails for video files.
+
+The Video Thumbnail Plugin provides the ability to generate thumbnails for video files stored in the Content Repository.
+
+It is an implementation of both the `PostCreateStoragePlugin` and `PostUpdateStoragePlugin` interfaces. When installed, it is invoked by the Catalog Framework immediately after a content item has been created or updated by the Storage Provider.
+
+This plugin uses a custom 32-bit LGPL build of https://ffmpeg.org/[FFmpeg] (a video processing program) to generate thumbnails. When this plugin is installed, it places the FFmpeg executable appropriate for the current operating system in `<${branding}_HOME>/bin_third_party/ffmpeg`. When invoked, this plugin runs the FFmpeg binary in a separate process to generate the thumbnail. The `<${branding}_HOME>/bin_third_party/ffmpeg` directory is deleted when the plugin is uninstalled.
+
+[NOTE]
+====
+Prebuilt FFmpeg binaries are provided for Linux, Mac, and Windows only.
+====
+
+===== Installing the Video Thumbnail Plugin
+
+The the Video Thumbnail Plugin is not installed by default with a standard installation.
```

Comment:
❗️ It looks like this is installed by default now. https://github.com/codice/ddf/blob/dfbce3ca3636bb80c98e5c2271e917686e5dd8d9/catalog/catalog-app/src/main/resources/features.xml#L94